docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -237,6 +238,7 @@ Json Parameters:
 -   **CpusetCpus** - String value containing the `cgroups CpusetCpus` to use.
 -   **CpusetMems** - Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
 -   **BlkioWeight** - Block IO weight (relative weight) accepts a weight value between 10 and 1000.
+-   **BlkioWeightDevice** - Block IO weight (relative weight for device) in the form of:        `"BlkioWeightDevice": ["device_path:weight"]
```

Comment:
Doesn't seem like this is the API anymore? Seems like this should be `[{"Path": "device_path", "Weight": weight}]`